db/migrate/20201111212135_add_offers_response_model.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class AddOffersResponseModel < ActiveRecord::Migration[6.0]
+  def change
+    create_table :offer_responses do |t|
+      t.references :offer, foreign_key: true, index: true
+      t.string :intended_state, null: false
+      t.boolean :interested_in_offers
```

Comment:
Not sure I understand this. If it's meant to reflect whether the response is generally positive, can it just be `interested`? Or, can it be implicit based on whether a `rejection_reason` is specified?

Reply:
It's definitely confusing.

This field won't coexist with `rejection_reasons`.

It's meant to indicate that the consignor is _still interested in consigning with Artsy_, but not interested in taking the current offer. 

But maybe you're right that `interested` is a better way to capture that.